### ISSUE TITLE
Update Support Doc Links in Checkout

### ISSUE THREAD

**Issue by donalirl** [Automattic/wp-calypso#81500]
### Quick summary

On the checkout screen, we link to [this support doc](https://wordpress.com/support/manage-purchases/) in three places. This link will soon change. The new links are:

how your subscription works: https://wordpress.com/support/manage-purchases/automatic-renewal/
how to cancel: https://wordpress.com/support/manage-purchases/cancel-a-purchase/
refunds: https://wordpress.com/support/refunds/

### Steps to reproduce

1. Add any purchase to cart on WordPress.com, such as a plan: https://wordpress.com/plans/ 
2. Proceed to checkout
3. View the links under Billing Information → Pick a payment method

<img width="610" alt="Screenshot 2023-09-07 at 9 28 04 am" src="https://github.com/Automattic/wp-calypso/assets/30727666/71b42e3e-ece1-4989-a58e-9d418a7b965d">

The text is:

>You agree to our [Terms of Service](https://wordpress.com/tos/) and authorize your payment method to be charged on a recurring basis until you cancel, which you can do at any time. You understand [how your subscription works](https://wordpress.com/support/manage-purchases/#automatic-renewal) and [how to cancel](https://wordpress.com/support/manage-purchases/#cancel-a-purchase).
>
>You understand that [refunds](https://wordpress.com/support/manage-purchases/#refund-policy) are limited to 14 days after purchase or renewal for non-domain products with yearly subscriptions.

### Impact

All

### Available workarounds?

No but the platform is still usable

### Platform (Simple and/or Atomic)

Simple, Atomic

### Logs or notes

The reason for the change is that the [Manage Purchases doc](https://wordpress.com/support/manage-purchases/) is too long and difficult to follow. We are breaking it up into smaller, more focused docs.

The new docs are live, but I will hold off on removing the information in the original doc until the Calypso links are updated.

- [x] Is it possible to check if `https://wordpress.com/support/manage-purchases/` is used elsewhere in Calypso that needs updating?

**Comment by cuemarie** [comment]
📌 ACTIONS
- Feature request kept
- FYI @Automattic/shilling

**Comment by jjchrisdiehl** [comment]
> Is it possible to check if https://wordpress.com/support/manage-purchases/ is used elsewhere in Calypso that needs updating?

If you go to https://github.com/Automattic/wp-calypso you can [type in the URL string into the search bar](https://github.com/search?q=repo%3AAutomattic%2Fwp-calypso%20https%3A%2F%2Fwordpress.com%2Fsupport%2Fmanage-purchases%2F&type=code) at the top to search the full Calypso repository.

Let me know if you need any other manage purchases links removed or change, I'll get a PR up for this soon.

**Comment by jjchrisdiehl** [comment]
Will mark as resolved once this PR is merged https://github.com/Automattic/wp-calypso/pull/81659

**Comment by donalirl** [comment]
> If you go to https://github.com/Automattic/wp-calypso you can [type in the URL string into the search bar](https://github.com/search?q=repo%3AAutomattic%2Fwp-calypso%20https%3A%2F%2Fwordpress.com%2Fsupport%2Fmanage-purchases%2F&type=code) at the top to search the full Calypso repository.

Nice, thanks @jjchrisdiehl! Did not know that. 

[I noticed a few more](https://github.com/search?q=repo%3AAutomattic%2Fwp-calypso+%2Fmanage-purchases%2F&type=code) when searching for `/manage-purchases/` instead of the full URL that it sometimes uses `${ root }/manage-purchases/#automatic-renewal` for example. 

Looking at the list, these links should change:

https://wordpress.com/support/manage-purchases/#refund-policy → https://wordpress.com/support/refunds/#refund-periods

https://wordpress.com/support/manage-purchases/#upgrade-credit → https://wordpress.com/support/manage-purchases/upgrade-your-plan/#upgrade-credit

https://wordpress.com/support/manage-purchases/#automatic-renewal → https://wordpress.com/support/manage-purchases/automatic-renewal/

https://wordpress.com/support/manage-purchases/#cancel-a-purchase → https://wordpress.com/support/manage-purchases/cancel-a-purchase/

Basically, any link to /support/manage-purchases/ is fine but any jump links to specific sections of /support/manage-purchases/# have a new location.

**Comment by jjchrisdiehl** [comment]
I believe I've added the additional cases you've noted, but we'll need to test them.

You can see the existing changes here https://github.com/Automattic/wp-calypso/pull/81659/files

For instance the auto-renewal link above that uses `{ root }` is related to the success message on the checkout thank you page after a successful autorenewal completes `'Success! You renewed %(productName)s. {{a}}Learn more about renewals{{/a}}'`.

Though I'm not quite sure how to trigger the above one for testing. Maybe @sirbrillig knows.

And the 100 Year Plan has a variation on the usual ToS component found at the bottom of checkout, this currently looks like the following on my sandbox (note the url in the bottom right):

<img width="898" alt="image" src="https://github.com/Automattic/wp-calypso/assets/16580129/42b250df-406b-45ea-bc72-89bc3d86101a">

**Comment by jjchrisdiehl** [comment]
Ah found it, the one link is when you renew a plan that _has a payment method attached to it_:

![image](https://github.com/Automattic/wp-calypso/assets/16580129/32c19bbc-fd7d-4d87-9cf8-3341b2788341)

All is tested now and I'll go ahead and merge this.

If you find any more links, feel free to reopen this issue.

**Comment by donalirl** [comment]
Thank you @jjchrisdiehl !
